.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+Please specify the following:
+
+1. A link to the original issue using `#` selector.
```

Comment:
Where is this link goes? Title, description or I can just link PR with issue i github?

Reply:
I've added, please check. I'm not sure I understood you correctly

Reply:
I mean does this link have to be in PR title or description?